src/include/timestamp_inline.h

```diff
@@ -181,25 +185,29 @@
     } while (0)
 
 /* Merge an aggregated time window into another - choosing the most conservative value from each. */
-#define WT_TIME_AGGREGATE_MERGE(session, dest, source)                                        \
-    do {                                                                                      \
-        WT_ASSERT(session, (dest)->init_merge == 1);                                          \
-        (dest)->newest_start_durable_ts =                                                     \
-          WT_MAX((dest)->newest_start_durable_ts, (source)->newest_start_durable_ts);         \
-        (dest)->newest_stop_durable_ts =                                                      \
-          WT_MAX((dest)->newest_stop_durable_ts, (source)->newest_stop_durable_ts);           \
-        (dest)->oldest_start_ts = WT_MIN((dest)->oldest_start_ts, (source)->oldest_start_ts); \
-        (dest)->newest_txn = WT_MAX((dest)->newest_txn, (source)->newest_txn);                \
-        (dest)->newest_stop_ts = WT_MAX((dest)->newest_stop_ts, (source)->newest_stop_ts);    \
-        (dest)->newest_stop_txn = WT_MAX((dest)->newest_stop_txn, (source)->newest_stop_txn); \
-        /*                                                                                    \
-         * Aggregation of newest transaction is calculated from both start and stop           \
-         * transactions. Consider only valid stop transactions.                               \
-         */                                                                                   \
-        if ((dest)->newest_stop_txn != WT_TXN_MAX)                                            \
-            (dest)->newest_txn = WT_MAX((dest)->newest_txn, (dest)->newest_stop_txn);         \
-        if ((source)->prepare != 0)                                                           \
-            (dest)->prepare = 1;                                                              \
+#define WT_TIME_AGGREGATE_MERGE(session, dest, source)                                            \
```

Comment:
Should we change this into a function to improve readability?

Reply:
I want to do this change in a different ticket, as this macro is called in many locations.